README.md

```diff
@@ -1,4 +1,4 @@
-[![License](https://img.shields.io/badge/License-Apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) [![Community](https://img.shields.io/badge/Join-Community-blue.svg)](https://developer.ibm.com/callforcode/racial-justice/get-started/)
+[![License](https://img.shields.io/badge/License-Apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0) [![Community](https://img.shields.io/badge/Join-Community-blue.svg)](https://callforcode.org/slack) [![Hacktoberfest](https://img.shields.io/badge/Celebrate-Hacktoberfest-orange.svg)](https://call-for-code-for-racial-justice.github.io/Hacktoberfest/#/?id=main)
```

Comment:
(https://img.shields.io/badge/Join-Community-blue.svg)]

changed from 
(https://developer.ibm.com/callforcode/racial-justice/get-started/) to
(https://callforcode.org/slack)

This shield change go to the call for code slack larger community and not the call for code for racial justice nor five/fifths voter.  

Why the change from the getting started, instead of the getting started linking to the various levels of programs? 

I would expect the community to be built around the cause and not the resource ( I want to converse with five-fifths voter, or on racial justice, not general code?) so I would expect for slack to end up at https://callforcode.slack.com/archives/C01CDLTU015 #racial-justice-five-fifths-voter or 
https://callforcode.slack.com/archives/C01D7MS0G0K #racial-justice-general

Is there a way for https://callforcode.org/slack to deep link the slack channels for redirect?
Otherwise, leave as the getting started, or to another md page in five-fifths voter that has the existing link plus explanation of how to reach the right channel?